terraform/modules/hub/hub_policy.tf

```diff
@@ -165,3 +249,46 @@ module "policy_can_connect_to_ingress_for_metadata" {
   source_sg_id      = module.policy_ecs_asg.instance_sg_id
   destination_sg_id = aws_security_group.ingress.id
 }
+
+module "policy_fargate_can_connect_to_config_fargate_v2" {
+  source = "./modules/microservice_connection"
+
+  source_sg_id      = module.policy_fargate.task_sg_id
+  destination_sg_id = module.config_fargate_v2.lb_sg_id
+}
+
+module "policy_fargate_can_connect_to_saml_engine" {
```

Comment:
should be going to saml_engine_fargate?

Reply:
We don't need this change. hub_policy.tf already has `policy_can_connect_to_saml_engine_fargate`.

Reply:
not policy_fargate_can_connect_to_saml_engine_fargate though?

Reply:
Too many fargate words.